histomicsui/web_client/stylesheets/panels/metadataWidget.styl

```diff
@@ -0,0 +1,84 @@
+.h-metadata-widget
```

Comment:
There is still a lot of repetition in this file echoing values that are defined in Girder.  Further, I'd like to remove some of the white space so that single key-value pairs can appear on the same line.  One possibility would be to simplify this file and change it like so:
```
This seems to look okay on my system with Chrome or Firefox, but I'm open to changes.
.h-metadata-widget
  .s-panel-title-container
    .s-no-panel-toggle
      position relative

  .g-widget-metadata-header-key-value
    margin-left 5px
    font-size 10px
    text-overflow ellipsis
    overflow hidden
    max-width 100px
    display inline-block
    white-space nowrap
    margin-bottom 2px

  .g-widget-metadata-row
    min-height 30px
    margin 0

  .g-widget-metadata-key, .g-widget-metadata-value
    vertical-align top
    margin 0

  .g-json-editor
    width 100%
    margin 0
    overflow hidden

    .jsoneditor-menu
      height initial
      min-height 35px
    table.jsoneditor-search
      position initial

  .g-widget-metadata-value
    width 59%

  button.g-widget-metadata-add-button
    float none
    margin 0 5px 0 0

  .g-widget-metadata-container .g-widget-metadata-edit-button
    margin 2px 0 0 0
```